### PR TITLE
Feature/location page

### PR DESCRIPTION
**Pull request by ad2969**
> :warning: **Needed for UHN-MA-20 on the frontend**
## Added Features
- Added location **GET**ting
- Changed all ```lon``` to ```lng``` for consistency (I think this is a standard across many packages)
- Included notes in location **GET**ting and **POST**ing

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ app.post("/login", validateLogin(), user.loginUser);
 
 app.post("/users/:id/responders", middleware.checkToken, user.addResponders);
 
-app.post("/users/:id/location", middleware.checkToken, user.addLocation);
+app.post("/users/:id/location", middleware.checkToken, user.updateLocation);
```
Comment (Asiapenguin) [inline server.js]: If we are updating location, shouldnt it be a PUT/PATCH? I realize i have let that slip for the previous POST endpoint to be pushed to master. Also have you thought of how we are going to get the user's location when they signup?

Comment (ad2969) [reply server.js]: Yeah! I could definitely make it a PUT instead, I think I was just blindly following the previous patterns when I was doing this :zipper_mouth_face:

Regarding the user's location on signup, I've left it blank for now - I'm not too sure but I dont know about getting the user's location in the background? In the front end, the app automatically gets the user's location on every enter to the location page - but it doesnt save this until the user clicks on ```save```

Comment (Asiapenguin) [reply server.js]: Go ahead with the change on PUT.

We can do a similar thing on the sign up screen. Grab GPS location upon visiting and converting it to an address to show on the field on screen, then we allow the user to change the address if it is incorrect. When they press signup, we will save the lat lon along with the newly created user. Do you think this is possible?

Comment (ad2969) [reply server.js]: So do you mean we'll be displaying the user's address somewhere in the sign up screen as well?

Comment (Asiapenguin) [reply server.js]: yes

**Pull request merged**